### DISCUSSION TITLE
Getting non-public schemas visible in Flutterflow.?

### DISCUSSION THREAD

**Discussion by shivaabhiramreddy** [supabase/supabase#28636] [General]
Been trying to create an app using flutter flow and supabase. I have sorted the tables in different schemas for ease of handling and better management and added those non-public schemas in the exposed schemas list.

When I integrated the supabase to the flutter flow app I can only see the tables in public schema. I have tried everything I could find online. Is there anyway to get all exposed schemas visible on flutter flow without writing a lot of code.

**Comment by GaryAustin1** [comment]
I'm not that familiar with flutterflow, but if you are doing REST API class you have to set a header (two different ones depending on operation) to use a different schema than default public one.

https://docs.postgrest.org/en/v12/references/api/schemas.html#get-head

**Comment by Matignaciom** [comment]
Hello! Remember to make sure every time you add new tables or information in your schema update it from flutterflow (settings and supabase section) update or get schema and it will update, maybe that solves your problem.

Regarding tables such as Auth, etc. Those are not visible there, only the public ones are visible.
